packages/eas-cli/src/commands/update/republish.ts

```diff
@@ -57,6 +62,15 @@ export default class UpdateRepublish extends EasCommand {
       description: 'Update group ID to republish',
       exclusive: ['branch', 'channel'],
     }),
+    'destination-channel': Flags.string({
+      description:
+        'Channel name to select a branch to republish to if republishing to a different branch',
+      exclusive: ['destination-branch'],
+    }),
+    'destination-branch': Flags.string({
+      description: 'Branch name to republish to if republishing to a different branch',
+      exclusive: ['destination-channel'],
+    }),
```

Comment:
Question for reviewers: should we add `hidden: true` to these?

Reply:
If it'd be useful, you can add `hidden: true` and I can dogfood it later this week. Else I'm fine with just releasing it.